### PR TITLE
[MRG] Clarified the thresholding terminology by removing the "level" term

### PR DESCRIPTION
**Pull request by bthirion**

### CODE REVIEW

**Comment by bthirion** [comment]
Addresses #280

**Comment by bthirion** [comment]
@kchawla-pi can you take a look ? Thx

File: examples/01_tutorials/plot_single_subject_single_run.py [hunk 1]
```diff
@@ -205,12 +205,12 @@
 # statistical validity of the procedure: here we used an arbitrary
 # threshold of 3.0 but the threshold should provide some guarantees on
 # the risk of false detections (aka type-1 errors in statistics). One
-# first suggestion is to control the false positive rate (fpr) at a
-# certain level, e.g. 0.001: this means that there is.1% chance of
-# declaring active an inactive voxel.
+# first suggestion is to control the false positive rate (fpr, denoted
```
Comment (kchawla-pi) [inline examples/01_tutorials/plot_single_subject_single_run.py]: One first suggestion? That is a weird statement. Maybe we mean One suggestion here?  
denoted by alpha will be better than denoted alpha

File: examples/01_tutorials/plot_single_subject_single_run.py [hunk 2]
```diff
@@ -205,12 +205,12 @@
...
+# alpha) at a certain level, e.g. 0.001: this means that there is .1% chance
+# of declaring active an inactive voxel.
```
Comment (kchawla-pi) [inline examples/01_tutorials/plot_single_subject_single_run.py]: of declaring an inactive voxel, active.

File: examples/01_tutorials/plot_single_subject_single_run.py [hunk 3]
```diff
@@ -205,12 +205,12 @@
 # statistical validity of the procedure: here we used an arbitrary
 # threshold of 3.0 but the threshold should provide some guarantees on
 # the risk of false detections (aka type-1 errors in statistics). One
-# first suggestion is to control the false positive rate (fpr) at a
-# certain level, e.g. 0.001: this means that there is.1% chance of
-# declaring active an inactive voxel.
+# first suggestion is to control the false positive rate (fpr, denoted
+# alpha) at a certain level, e.g. 0.001: this means that there is .1% chance
```
Comment (kchawla-pi) [inline examples/01_tutorials/plot_single_subject_single_run.py]: Making it 0.1% instead of .1% reduces the probability of misreading this.

**Comment by kchawla-pi** [comment]
I'll make another pass at this in the morning. There is something I wanna look at again.

**Comment by kchawla-pi** [comment]
Whatever it was that caught my eye, I don't see it now. Seems good to me.

**Comment by bthirion** [comment]
Merging now.

**Pull request merged**